types/mini-css-extract-plugin/mini-css-extract-plugin-tests.ts

```diff
@@ -49,14 +49,14 @@ configuration = {
 
 configuration = {
     // ...
-    plugins: [new MiniCssExtractPlugin()],
+    plugins: [new MiniCssExtractPlugin({})],
 };
 
 configuration = {
     // ...
     plugins: [
         new MiniCssExtractPlugin({
-            filename: 'styles.css',
+            filename: ({ chunk }) => `${chunk.name.replace('/js/', '/css/')}.css`,
```

Comment:
There is now no longer a test with `filename` with a value of `string`? Probably would be good to test both `string` and the function format?

Reply:
no, there are in other blocks before and after, e.g:
https://github.com/peterblazejewicz/DefinitelyTyped/blob/update/mini-css-extract-plugin/types/mini-css-extract-plugin/mini-css-extract-plugin-tests.ts#L69

Reply:
ah my bad, I thought I expanded everything in the diff view but missed that, sorry about that!